docs/guides/data_viewers/tableau.md

```diff
@@ -29,35 +28,100 @@ CREATE VIEW h2oai AS (
+for Tableau to use.
+For now, we recommend using the latest bleeding edge JDBC driver (0.8.2) as a number of fixes have been made 
+for time compatibility.
+Note that Tableau (both Desktop and Server versions) need to be restarted any time you add or modify drivers.
+
+## Driver Links
+
+The links here are for a recent version of the JDBC driver that is compatible with Tableau.
+If you wish to connect to a database file,
+you will need to make sure the file was created with a file-compatible version of DuckDB.
+Also, check that there is only one version of the driver installed as there are multiple filenames in use.
+
+* MacOS Desktop.
+  * [Download](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094935) the DuckDB JDBC driver. This will a file called `duckdb_jdbc-osx-universal.jar`.
+  * Copy it to the `~/Library/Tableau/Drivers/` folder.
+* Windows Desktop.
+  * [Download](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094936) the DuckDB JDBC driver. This will be a file called `duckdb_jdbc.jar`.
+  * Copy it to the `C:\Program Files\Tableau\Drivers` directory. 
+* Linux Server.
+  * Download the DuckDB JDBC driver appropriate for your architecture, either
+    * [AMD64](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094934) or
+    * [AARCH64](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094932).
+  * Copy it to the `/opt/tableau/tableau_driver/jdbc`.
+* Windows Server.
+  * [Download](https://github.com/duckdb/duckdb/suites/14840102996/artifacts/845094936) the DuckDB JDBC driver. This will be a file called `duckdb_jdbc.jar`.
+  * Copy it to the `C:\Program Files\Tableau\Drivers` directory. 
+
+# Using the Postgres Dialect
+
+If you just want to do something simple, you can try connecting directly to the JDBC driver 
+and using Tableau-provided Postgres dialect.
+
+1. Create a DuckDB file containing your views and/or data.
+2. Launch Tableau
+3. Under Connect > To a Server > More… click on “Other Databases (JDBC)” This will bring up the connection dialogue box. For the URL, enter jdbc:duckdb:/User/username/path/to/database.db. For the Dialect, choose PostgreSQL. The rest of the fields can be ignored:
+
+![tableau-postgres](/images/guides/tableau/tableau-osx-jdbc.png)
+
+# Installing the Tableau DuckDB Connector
+
+While it is possible to use the Tableau-provided Postgres dialect to communicate with the DuckDB JDBC driver,
+we strongly recommend using the [DuckDB "taco" connector](https://github.com/hawkfish/duckdb-taco).
+This connector has been fully tested against the Tableau dialect generator 
+and [is more compatible](https://github.com/hawkfish/duckdb-taco/blob/main/tableau_connectors/duckdb_jdbc/dialect.tdd)
+than the provided Postgres dialect.
+
+The documentation on how to install and use the connector is in its repository,
+but essentially you will need the `duckdb_jdbc.taco` file from the `duckdb-taco/packaged-connector` directory.
```

Comment:
@hawkfish Are we able to provide a download link in the documentation such that users can download the taco directly without needing to navigate to the GitHub page?

Reply:
Good point - I was being lazy and not figuring out how to force GitHub to give me the right link...